keyboards/exclusive/e6_rgb/keymaps/allleds/rules.mk

```diff
@@ -0,0 +1 @@
+RGB_MATRIX_ENABLE = IS31FL3733# Use RGB matrix
```

Comment:
It would be good to have a readme file for this keymap to note that it uses both RGB Underglow *and* per-key RGB, and how.

Reply:
add readme file for the usage